modules/runner/gateway_maticbor/runners.go

```diff
@@ -0,0 +1,46 @@
+package gateway_maticbor
+
+import (
+	"errors"
+
+	"github.com/marlinprotocol/ctl2/modules/runner"
+)
+
+func GetRunnerInstance(runnerId string, version string, storage string, runnerData interface{}, skipRunnerData bool, skipChecksum bool, instanceId string) (runner.Runner, error) {
+	switch runnerId {
+	case "linux-amd64.supervisor.runner02":
```

Comment:
Why are we using `runner02` on a new project? we never had `runner01`. Please name the newest release and current runner implementation as `runner01`